README.md

```diff
@@ -1 +1,111 @@
-# Seafood-capstone-project
\ No newline at end of file
+<div align="center">
+
+  <img src="./src/Asset/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h1><b>Seafood Capstone</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is a capstone project for Microverse, where you can see meals that we got from the API, on this website you are able to like the meals, comment the meals, and add reservation for the meals.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Jest
+- API
+
+### Key Features
+
+- Getting meal cards from the API
+- Liking the meals with the counter
+- Adding comments on it
+- Adding reservations for the meals
+
+## Live Demo
+
+[Live Demo](https://zeff96.github.io/Seafood-capstone-project/dist/)
+
+## Getting Started
+
+### Clone this [repository](https://github.com/zeff96/Seafood-capstone-project) to your desired folder.
+
+### Run this command to install dependencies
+
+> npm install
+
+### Run this command to run server
+
+> npm start
+
+### Run this command to build your webpack
+
+> npm run build
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if the `running build` command comes before `running start` command. This is because the `build for production` must be built before you run your app in the sever. i.e.
```
...
### Run this command to build your webpack

> npm run build

### Run this command to run server

> npm start
...
```